js/clipboard.js

```diff
@@ -59,6 +59,34 @@ $(function() {
             target.parent('.copy_to_clipboard_wrapper').addClass('copyfail');
         }
     });
+
+    /**
+     * For each input of type text,
+     * if the input get a paste event, the text is trimmed before being pasted
+     */
+    $(document).on("paste", "input[type='text']", function (event) {
+        // Cancel the default action
+        event.preventDefault();
+
+        // Get the text from the clipboard
+        var text = event.originalEvent.clipboardData.getData('text/plain');
+
+        // Get the actual text in the input
+        var actual = $(this).val();
+
+        /**
+         * if actual value is empty, trim the text
+         * else add the text to the actual value
+         */
+        if (actual === '') {
+            text = text.trim();
+        } else {
+            text = actual + text;
+        }
+
+        // Paste the text in the input
+        $(this).val(text);
+    });
```

Comment:
To test:
```suggestion
    $(document).on("paste", "input[type='text']", function (event) {
        if ($(this).val() === '') {
            event.preventDefault();
            var pastedData = event.originalEvent.clipboardData || window.clipboardData;
            var pastedText = pastedData.getData('text');
            $(this).val(pastedText.trim());
        }
    });
```

Reply:
Applied in [4dea0c1](https://github.com/glpi-project/glpi/pull/16431/commits/4dea0c1691c06d3d41a502bdbed137bbc94c9fbc)